raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -39,14 +39,14 @@ def clear_if_finalized(iteration: TransitionResult) -> TransitionResult:
 def transfer_exists(
         payment_state: InitiatorPaymentState,
         secrethash: SecretHash,
-):
+) -> bool:
     return secrethash in payment_state.initiator_transfers
 
 
 def cancel_other_transfers(
         payment_state: InitiatorPaymentState,
         unlocked_secrethash: SecretHash,
-):
+) -> None:
```

Comment:
This shouldn't be necessary. The function is already typed and no return type annotation implies `None`.

Reply:
Yes, but I'm pretty sure we agreed to be explicit for this.